nodewatcher/modules/monitor/sources/snmp/apps.py

```diff
@@ -0,0 +1,7 @@
+from django import apps
+
+
+class SNMPDConfig(apps.AppConfig):
```

Comment:
I think this should be named `SNMPConfig` as the module (and the protocol) is SNMP. Also, fix above in `__init__.py`.